connectors/src/connectors/slack/bot.ts

```diff
@@ -300,6 +316,10 @@ async function botAnswerMessage(
       title: null,
       visibility: "unlisted",
       message: messageReqBody,
+      contentFragment: {
+        title: "Slack conversation",
```

Comment:
`Slack thread messages` will be more appropriate I think as initial fragment and follow-up fragment?

Reply:
done thanks.